mondrian/src/main/java/mondrian/rolap/SqlConstraintUtils.java

```diff
@@ -1674,6 +1674,12 @@ public static String constrainLevel(
                 // as the key column
                 int bitPos = column.getBitPosition();
                 AggStar.Table.Column aggColumn = aggStar.lookupColumn(bitPos);
+
+                if(aggColumn == null){
+                    LOG.warn("Aggregate table: A constraint will not be generated because name column is not the same as key column");
```

Comment:
This message should be externalized in ``mondrianresource.xml``

Reply:
Agree, thanks!